src/index.css

```diff
@@ -31,17 +31,36 @@
   background: var(--clr-dark);
 }
 
+.testing-one {
+  /* background-color: green; */
+}
+
+.testing-two {
+  /* background-color: yellow; */
+}
```

Comment:
These look like unused code. It's bad coding hygiene to leave things like this in your files.

Reply:
I was testing some stuff with flexbox and getting the text and images on the left and the main board on the right. I'll delete it.